base_url/tests/test_abstract_url.py

```diff
@@ -118,3 +118,10 @@ def test_write_inactive(self):
         ) as mocked_redirect:
             my_partner.active = False
             mocked_redirect.assert_called_once()
+
+    def test_onchange_do_nothing(self):
+        # calling onchange should not fail and should not return an url
```

Comment:
@sebastienbeau You should add a tests for the oposite to ensure that an onchange on an existing record update the url

Reply:
In fact onchange are always played with NewID (even if the record exist). Regarding the case of testing the _generic_compute_automatic_url_key it's already done with the other test (that check the url generated automatically)